app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,36 @@
+class QuotesController < ApplicationController
+  def show
+    @quote = Quote.find_by(id: params[:id])
+    if @quote.nil?
+      render :error
```

Comment:
I think `find!` is deprecated now?
```
Quote.respond_to? :find!
=> false
```
Maybe I'm not seeing something but the difference in the error page rails shows between using `find`, `find_by`, and `find_by!` is very slight